client/disputes/index.tsx

```diff
@@ -240,6 +253,23 @@ export const DisputesList = (): JSX.Element => {
 				display: clickable( dispute.customer_country ),
 			},
 			details: { value: dispute.dispute_id, display: detailsLink },
+			action: {
+				value: '',
+				display: needsResponse ? (
+					<Button
+						variant={ 'secondary' }
+						href={ getDetailsURL( dispute.dispute_id, 'disputes' ) }
+					>
+						{ __( 'Respond', 'woocommerce-payments' ) }
+					</Button>
+				) : (
+					<Link
+						href={ getDetailsURL( dispute.dispute_id, 'disputes' ) }
+					>
+						{ __( 'See details', 'woocommerce-payments' ) }
+					</Link>
```

Comment:
In the mockups this isn't shown as an underlined link. IMO not having the underline here is not clear that it is clickable.